### PR TITLE
fix: modules time in selection area

### PR DESCRIPTION
**Pull request by saisilinus**
This fix auto-generates the total time of training modules in the module selection area

Closes ASKnetCommunity/Training#105, Closes ASKnetCommunity/Training#110

### CODE REVIEW

File: training.html [hunk 1]
```diff
@@ -103,7 +108,24 @@ <h2>{{ module.module.name }}</h2>
...
+                                    {% if module_duration_hours > 0 %}
+                                        {% if module_duration_hours == 1 %}
+                                            {{ module_duration_hours | append: ' ' }} Hour
+                                            {% else %}
+                                            {{ module_duration_hours | append: ' ' }} Hours
+                                        {% endif %}
+                                    {% endif %}
+                                    {% if module_duration_minutes > 0 %}
+                                        {% if module_duration_minutes == 1 %}
+                                            {{ module_duration_minutes }} Minute
```
Comment (timmwille) [inline training.html:122]:
```suggestion
                                            {{ module_duration_minutes }} min
```

File: training.html [hunk 2]
```diff
@@ -103,7 +108,24 @@ <h2>{{ module.module.name }}</h2>
...
+                                            {{ module_duration_hours | append: ' ' }} Hour
+                                            {% else %}
+                                            {{ module_duration_hours | append: ' ' }} Hours
+                                        {% endif %}
+                                    {% endif %}
+                                    {% if module_duration_minutes > 0 %}
+                                        {% if module_duration_minutes == 1 %}
+                                            {{ module_duration_minutes }} Minute 
+                                            {% else %}
+                                            {{ module_duration_minutes }} Minutes
```
Comment (timmwille) [inline training.html:124]:
```suggestion
                                            {{ module_duration_minutes }} min
```

File: training.html [hunk 3]
```diff
@@ -103,7 +108,24 @@ <h2>{{ module.module.name }}</h2>
...
                     <div class="module-meta">
                         <div class="module-meta">
-                            <p class="module-duration"><i class="fas fa-hourglass-half"></i>{{ module.module.duration | divided_by:60 }} Hours {{ module.module.duration | modulo:60 }} Minutes</p>
+                            {% if module_duration > 0 %}
+                                <p class="module-duration"><i class="fas fa-hourglass-half"></i>
+                                    {% if module_duration_hours > 0 %}
+                                        {% if module_duration_hours == 1 %}
+                                            {{ module_duration_hours | append: ' ' }} Hour
+                                            {% else %}
+                                            {{ module_duration_hours | append: ' ' }} Hours
```
Comment (timmwille) [inline training.html:117]:
```suggestion
                                            {{ module_duration_hours | append: ' ' }} h
```

**Comment by timmwille** [comment]
@saisilinus double check my requested changes or implement them in a better way without the `ifelse` approach (though I like it is possible for other context). Just `h` and `min` for one or multiple hours and minutes is fine. Apart from that it is fine and I will merge

**Review by timmwille — CHANGES_REQUESTED** [review]
small adjustment for minutes and hours just "h" and "min" is fine

**Comment by saisilinus** [comment]
It should be okay now @timmwille

**Review by timmwille — APPROVED** [review]
nice one, thank you @saisilinus

**Pull request merged**